### PR TITLE
Fix lint 

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Review by lmignon — APPROVED** [review]

File: purchase_manual_service_quantity/models/purchase_order_line.py [hunk 1]
```diff
@@ -2,8 +2,7 @@
 # Copyright 2016-2017 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models, _
-from odoo.exceptions import UserError
```
Comment (gurneyalex) [inline purchase_manual_service_quantity/models/purchase_order_line.py:6]: maybe we don't want to remove this import but to raise an exception in _inverse_qty_received... To be checked

**Comment by lmignon** [comment]
ping @adrienpeiffer

**Comment by gurneyalex** [comment]
@lmignon @adrienpeiffer I'll be happy to improve this PR to fix the UserError issue and get a green build, if you point me in the direction you prefer (i.e. exception raised or no exception)

**Comment by lmignon** [comment]
@gurneyalex Thank you for the proposal and the first fix. @adrienpeiffer is the author of this PR so I let him answer the question about the expected behaviour behind this UserError

**Comment by adrienpeiffer** [comment]
Actually, I don't remember about the UserError. But we run this code in production. So, for me we can remove the import and change the test. To be sure, @benwillig Any idea ? I know that you worked on this kind of things during my holidays.

**Comment by gurneyalex** [comment]
I just pushed another commit removing the offending test. This should make Travis Green and allow moving forward with the merging of this addon :smile_cat:

**Comment by gurneyalex** [comment]
lint errors are not related to this PR

**Comment by lmignon** [comment]
Thank you @gurneyalex !

**Comment by gurneyalex** [comment]
gahh I left one import in my PR, now unused...

**Pull request merged**